.github/workflows/tests.yml

```diff
@@ -6,6 +6,9 @@ on:
       - dev
 jobs:
   tests:
```

Comment:
`tests` was set as required to pass to allow PR to be merged. Since it's a matrix now, I've set all combinations as required. 
[Protection rules for dev](https://github.com/Android-Guild/Picover-KMP/settings/branch_protection_rules/43926365)
Probably we would need something like: https://github.com/orgs/community/discussions/2682 - to be considered

Reply:
second link doesn't work 😄

Reply:
Fixed